share/spice/hex/content.handlebars

```diff
@@ -0,0 +1,8 @@
+<p><strong>{{releases.0.version}}</strong> is the latest of <strong>{{releases.length}}</strong> releases and it was published <em>{{updated_at}}</em>.</p>
```

Comment:
@moollaza Thanks for your advice, already applied your suggestion and bookmarked the link :bowtie: 

May I ask if you have some guidelines for JavaScript code? I have not seen any Task Manager (e.g. [Gulp](http://gulpjs.com/)) in the `zeroclickinfo-spice` project to run a static code analysis tool (e.g. JSHint) or verify that our JS code complies with the DuckDuckGo's JS style guide, using something like [JSCS](http://jscs.info/).

Thanks in advance for your help.